File: string-handling/substrings/visual-basic/strip_whitespace.vb
' Raspberry Pi Foundation
' Usage licensed under CC BY-NC-SA 4.0
' Note: This file is designed to be copied out and compiled on your machine.
' To run this file you need to:
' 1. Copy the contents
' 2. Paste them into the Visual Basic IDE of your choice (Visual Studio, for example)
' 3. Compile the program
' 4. Run the program

Module Program

    ' The Main subroutine is the default entry point for a VB program
    Sub Main()
        StripWhitespace()
    End Sub

    Sub StripWhitespace()
        Dim myString As String = "     Ada Computer Science  " & vbLf
        Dim strippedString As String = myString.Trim()
        Console.WriteLine(strippedString)
    End Sub


End Module

File: string-handling/substrings/python/strip_whitespace.py
# Raspberry Pi Foundation
# Developed as part of Ada Computer Science
# 
# Usage licensed under CC BY-NC-SA 4.0

def strip_whitespace():
    """Demonstrates stripping whitespace from the start and end of a string"""
    my_string = "     Ada Computer Science  \n"
    stripped_string = my_string.strip()
    print(stripped_string)
    

# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == "__main__":
    strip_whitespace()
